src/screens/ENSConfirmRegisterSheet.tsx

```diff
@@ -295,7 +295,8 @@ export default function ENSConfirmRegisterSheet() {
             goToProfileScreen();
           }}
           isSufficientGas={Boolean(
-            registrationCostsData?.isSufficientGasForStep
+            registrationCostsData?.isSufficientGasForRegistration &&
```

Comment:
extend registration gas validation was working correctly, but we weren't taking in account the rent price for the amount of years the users wanted to register for. With this change we validate enough ETH to do the registration extension, taking in account gas and also the ETH value sent with the transaction